src/routes/github/configuration/github-configuration-post.ts

```diff
@@ -32,6 +36,30 @@ export const GithubConfigurationPost = async (req: Request, res: Response): Prom
 			});
 		return;
 	}
+	if (await booleanFlag(BooleanFlags.ENABLE_GITHUB_SECURITY_IN_JIRA, installation.jiraHost)) {
+		await submitSecurityWorkspaceToLink(installation.jiraHost, gitHubInstallationId, gitHubAppId, req.log);
```

Comment:
I'm curious if this should be inside `verifyAdminPermsAndFinishInstallation`? Because there could be other ways when the subscription got created (e.g. "deferred install flow" - the subscription is created once GitHub admin approves it)